contrib/multivariate-tada/genunivdata.py

```diff
@@ -0,0 +1,50 @@
+import pandas as pd
+import torch
+
+#Simulates univariate data based on demo mutation rates in original TADA code
+
+def genUnivData(gamma, q, N_family, N_case, N_control, mut_1, mut_2):
```

Comment:
This method needs a docstring too, preferably a multi-line one that glosses the parameters and return value :)